pom.xml

```diff
@@ -10,9 +10,27 @@
     </parent>
     <artifactId>sirius-kernel</artifactId>
     <name>SIRIUS kernel</name>
-    <version>DEVELOPMENT-SNAPSHOT</version>
+    <version>12.0-rc27adbe1</version>
```

Comment:
Versions are automatically built and released via our CI server - please undo the changes in the pom.xml and we can merge and build a release.